openstack/dcs/v2/tags/requests.go

```diff
@@ -0,0 +1,79 @@
+package tags
+
+import "github.com/chnsz/golangsdk"
+
+var RequestOpts = golangsdk.RequestOpts{
+	MoreHeaders: map[string]string{"Content-Type": "application/json", "X-Language": "en-us"},
+}
+
+//ResourceTag is in key-value format
+type ResourceTag struct {
+	Key string `json:"key" required:"ture"`
+	// Value is not required, but it must be included in the parameter.
+	Value string `json:"value"`
+}
+
+type createTagsOpts struct {
+	Action string        `json:"action" required:"true"`
+	Tags   []ResourceTag `json:"tags,omitempty"`
+}
+
+func Create(c *golangsdk.ServiceClient, instanceID string, tags []ResourceTag) error {
+	opts := createTagsOpts{
+		Action: "create",
+		Tags:   tags,
+	}
+	b, err := golangsdk.BuildRequestBody(opts, "")
+	if err != nil {
+		return err
+	}
+
+	_, err = c.Post(actionURL(c, instanceID), b, nil, &golangsdk.RequestOpts{
+		OkCodes:     []int{204},
+		MoreHeaders: RequestOpts.MoreHeaders,
+	})
+	return err
+}
+
+func Delete(c *golangsdk.ServiceClient, instanceID string, tags []ResourceTag) error {
+	opts := createTagsOpts{
+		Action: "delete",
+		Tags:   tags,
+	}
+	b, err := golangsdk.BuildRequestBody(opts, "")
+	if err != nil {
+		return err
+	}
+
+	_, err = c.Post(actionURL(c, instanceID), b, nil, &golangsdk.RequestOpts{
+		OkCodes:     []int{204},
+		MoreHeaders: RequestOpts.MoreHeaders,
+	})
+	return err
+}
+
+func Get(c *golangsdk.ServiceClient, instanceID string) ([]ResourceTag, error) {
+	var rst golangsdk.Result
+	_, err := c.Get(resourceURL(c, instanceID), &rst.Body, &golangsdk.RequestOpts{
+		MoreHeaders: RequestOpts.MoreHeaders,
+	})
+	if err == nil {
+		var r []ResourceTag
+		rst.ExtractInto(&r)
+		return r, nil
+	}
+	return nil, err
+}
+
+func List(c *golangsdk.ServiceClient, instanceID string) (*ResourceTagList, error) {
```

Comment:
It seems that Get and List use the same URI, what's the difference between them?

Reply:
the List method should use listURL, I have modified it